<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>420</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Myriad Pro</family>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>401</width>
      <height>371</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Myriad Pro</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Main set-up</string>
    </property>
    <widget class="QPushButton" name="task_stop">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>STOP</string>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>381</width>
       <height>43</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="valve1">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Valve1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="valve2">
        <property name="text">
         <string>Valve2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="valve3">
        <property name="text">
         <string>Valve3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="valve4">
        <property name="text">
         <string>Valve4</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>141</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Mouse ID</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <widget class="QLineEdit" name="mouse_id"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>184</y>
       <width>381</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox_9">
        <property name="title">
         <string>Total trial</string>
        </property>
        <widget class="QLineEdit" name="trial_total">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>81</width>
           <height>39</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_7">
        <property name="title">
         <string>Current trial</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLCDNumber" name="trial_current"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_8">
        <property name="title">
         <string>In block trial</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QLCDNumber" name="trial_inblock"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="task_start">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>START</string>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>381</width>
       <height>81</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QGroupBox" name="groupBox_6">
        <property name="title">
         <string>Current location</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLCDNumber" name="position">
           <property name="enabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_12">
        <property name="title">
         <string>Task time</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLCDNumber" name="task_timer">
           <property name="enabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_13">
        <property name="title">
         <string>Laser state</string>
        </property>
        <widget class="QLineEdit" name="laser_state">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>30</y>
           <width>71</width>
           <height>39</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_11">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>171</width>
      <height>371</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Myriad Pro</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Laser set-up</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>102</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>High time (ms)</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="laser_hightime">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>42</y>
          <width>61</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>98</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Low time (ms)</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="laser_lowtime">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>43</y>
          <width>61</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>107</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Laser on sensor</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="laser_on">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>42</y>
          <width>61</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="title">
        <string/>
       </property>
       <widget class="QLineEdit" name="laser_off">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>42</y>
          <width>61</width>
          <height>26</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>108</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Laser off sensor</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>20</width>
      <height>371</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>610</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
